src/new-client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -145,48 +146,70 @@ const DATE_FORMAT = "dd-MMM-yyyy"
 
-export const TradesGrid: React.FC = () => {
-  const trades = useTableTrades()
+interface RowParam {
+  index: number
+  style: unknown
+}
+
+const MainTable: React.FC = ({ children }) => {
   const tableSort = useTableSort()
   return (
-    <TableWrapper>
-      <Table>
-        <TableHead>
-          <TableHeadRow>
-            <StatusIndicatorSpacer />
-            {colConfigs.map(({ field, headerName, numeric }) => (
-              <TableHeadCell
-                key={field}
-                numeric={numeric}
-                onClick={() => sortFieldSelections$.next(field)}
-              >
-                {headerName}
-                {tableSort.field === field ? (
-                  tableSort.direction === "ASC" ? (
-                    <FaLongArrowAltUp />
-                  ) : (
-                    <FaLongArrowAltDown />
-                  )
+    <Table>
+      <TableHead>
+        <TableHeadRow>
+          <StatusIndicatorSpacer />
+          {colConfigs.map(({ field, headerName, numeric }) => (
+            <TableHeadCell
+              key={field}
+              numeric={numeric}
+              onClick={() => sortFieldSelections$.next(field)}
+            >
+              {headerName}
+              {tableSort.field === field ? (
+                tableSort.direction === "ASC" ? (
+                  <FaLongArrowAltUp />
                 ) : (
-                  <span className="spacer" />
-                )}
-              </TableHeadCell>
-            ))}
-          </TableHeadRow>
-        </TableHead>
-        <tbody>
-          {trades.map((trade) => (
-            <TableBodyRow key={trade.tradeId}>
-              <StatusIndicator status={trade.status} />
-              {colConfigs.map(({ field, numeric, valueFormatter }) => (
-                <TableBodyCell key={field} numeric={numeric}>
-                  {valueFormatter?.(trade[field]) ?? trade[field]}
-                </TableBodyCell>
-              ))}
-            </TableBodyRow>
+                  <FaLongArrowAltDown />
+                )
+              ) : (
+                <span className="spacer" />
+              )}
+            </TableHeadCell>
           ))}
-        </tbody>
-      </Table>
+        </TableHeadRow>
+      </TableHead>
+      {children}
+    </Table>
+  )
+}
+
+export const TradesGrid: React.FC = () => {
+  const trades = useTableTrades()
+
+  const Row = ({ index, style }: RowParam) => (
+    <TableBodyRow key={trades[index].tradeId}>
+      <StatusIndicator status={trades[index].status} />
+      {colConfigs.map(({ field, numeric, valueFormatter }) => (
+        <TableBodyCell key={field} numeric={numeric}>
+          {valueFormatter?.(trades[index][field]) ?? trades[index][field]}
+        </TableBodyCell>
+      ))}
+    </TableBodyRow>
+  )
+
+  return (
+    <TableWrapper>
+      <List
+        className="List"
+        height={150}
+        itemCount={trades.length}
+        itemSize={30}
```

Comment:
Do we know if it can accept units other than pixels??--specifically, `rem` values?

Reply:
I don't think it's doable since height attribute only accepts number type.